src/assisted_test_infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -599,6 +600,40 @@ def set_ipxe_url(self, network_name: str, ipxe_url: str):
         self.destroy_network(network)
         self.create_network(xml.toprettyxml())
 
+    def _create_dns_host_xml(self, api_vip: str, cluster_name: str):
+        """
+        Creates an XML objet for adding a DNS host to a libvirt network
```

Comment:
```suggestion
        Creates an XML object for adding a DNS host to a libvirt network
```

Reply:
Thanks for catching this! Fixed the spelling